src/common/exif.cc

```diff
@@ -1110,6 +1110,19 @@ int dt_exif_write_blob(uint8_t *blob, uint32_t size, const char *path, const int
       dt_remove_exif_keys(imgExifData, keys, n_keys);
     }
 
+    // remove the profile tables
```

Comment:
Agreed, I can do another fix for this. What would be the prefered approach: adding a new "dng_mode" argument to the function like for read, or determining the output format from the initial exif/metadata retrieved from the image?